assets/libraries/openapi/library.rego

```diff
@@ -29,6 +29,11 @@ incorrect_ref(ref, object) {
 	regex.match(references[object], ref) == false
 }
 
+is_security_scheme_http_basic(securitySchemes){
+    lower(securitySchemes.regularSecurity.type) == "http"
+    lower(securitySchemes.regularSecurity.scheme) == "basic"
```

Comment:
I think this is wrong, `regularSecurity` is the name of a `securityScheme`, there is nothing on specification refering to `regularSecurity` key.
Checking [component object's documentation](https://swagger.io/specification/#components-object), it is possible to say that `regularSecurity` (or any other name) is just a reference for the security scheme